Fitness/Models/ProBio.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  ProBio.swift
+//  Fitness
+//
+//  Created by Cameron Hamidi on 2/28/19.
+//  Copyright © 2019 Keivan Shahida. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+struct ProBio {
+    let id: String
+    let name: String
+    let profilePic: UIImage?
```

Comment:
Are these images downloaded from our servers or stored in the app's bundle? If the former, we should probably make these URLs. If the latter, we should be able to get away with using the swift struct pre-made init function.

Reply:
Since backend connectivity hasn't been implemented yet they're image assets, but when that changes I'll make them URLs